### ISSUE TITLE
As a FS Admin user I would like an easy way to get to the admin forms

### ISSUE THREAD

**Issue by sadlerw** [nciinc/fs-permit-platform#460]
## Acceptance criteria
- [ ] admin users see menu option in header that lets them navigate to christmas tree forms (/admin/christmas-trees/reports, season-dates, district-dates)
- [ ] non-admin users don't see the navigation
- [ ] admins without forests don't see the navigation

**Comment by sadlerw** [comment]
![image](https://user-images.githubusercontent.com/2058839/36393446-39ccc68c-1575-11e8-8fb6-411ff5b0ade0.png)

**Comment by sadlerw** [comment]
either have Christmas Trees Admin in the header route to a page with all reports on it, or show links to individual reports in the green bar?

**Comment by ASprinkle** [comment]
This story would include the link to the Google Form for editing other content on the page, correct?

**Comment by lauraGgit** [comment]
FYI, seems like the menu should be consistent with the FS USWDS fork menu style - https://nciinc.github.io/fs-fork-uswds-site/components/
![image](https://user-images.githubusercontent.com/5840989/36798046-8c0f0168-1c77-11e8-968b-cd7452c5e2ac.png)

**Comment by sadlerw** [comment]
<img width="1339" alt="screen shot 2018-03-09 at 12 42 37 pm" src="https://user-images.githubusercontent.com/2058839/37223869-7b3e4f3e-2397-11e8-8c77-94d4cfe0839e.png">
@tram @MelissaBraxton

**Comment by ASprinkle** [comment]
@sadlerw (slack is having connection issues on my end) As an FS Admin I would see those link options, but as a member of the public I would not, correct?
Can we title each option differently? "Generate Reports", "Change Season Dates", "Change Cutting Area Dates".  For the Christmas tree permits site link.. can we call it something else? "E-permit Site".
Can we design them so that they look like clickable button options/tabs?
Are we assuming that the FS admin would have to select their forest first in order to see the tabular options? Or since the Admin pages have options to select forests, would it be helpful to have the tabs also be available on the Christmas tree landing page?  As an FS admin, if I am on my Forests page, and then select a "generate report" tab, I would assume that I wouldn't have to select my forest again.
@MelissaBraxton @lauraGgit Can the google form to request changes be accessed somewhere here as well for FS Admins? maybe another tab or something that says "Request Website Edits" or something along those lines?  If I recall, the idea was to limit the FS Admins that can use that form to request changes - so maybe it's not an appropriate place for it.  Curious on yours and teams thoughts on that.

**Comment by sadlerw** [comment]
1. only fs admins see this (admin users that have forests configured in the allowed list)
2. I will change titles to your suggestions
3. The design matches the uswds so I think it's ok as is
4. no forest selection is needed prior to clicking
5. the menu is available on all pages on the site once and admin with forests is logged in
6. passing the current forest in the url to the report or other admin pages is certainly possible, but I'm not sure it's needed - the common pattern here is that you would select your forest again and it isn't that onerous an action (click radio button) and most users won't have to select at all because they will only have one forest
7. I think this would be a good place to link to the google edits form - +1

**Comment by sadlerw** [comment]
<img width="1348" alt="screen shot 2018-03-09 at 2 13 06 pm" src="https://user-images.githubusercontent.com/2058839/37227780-082c6384-23a4-11e8-9c51-a8c15d26d4a3.png">

**Comment by ASprinkle** [comment]
Thanks @sadlerw I appreciate the feedback. I am not convinced that my suggestions for the titles are the best - so certainly open to other ideas there.

**Comment by JaneZC** [comment]
@sadlerw What page will show when an admin clicks on the Christmas tree permits page? I am pretty sure we discussed this already so I am sorry I can't remember how someone will login and then click through the navigation and what they will see. I think I am following all the navigation items, except the first one.

**Comment by sadlerw** [comment]
/christmas-trees/forests is behind that link - if you look at the title on the page you can see they match

**Comment by MelissaBraxton** [comment]
Looking good! We should be consistent in how we refer to the "Christmas tree permits" page. On the 404, we call it the "Christmas tree information" page.

**Comment by sadlerw** [comment]
good catch @MelissaBraxton  - I fixed the language on the 404 page.  This is on staging for review, though I think we have conflicting styles on the mobile menu between the sidebar and the top mobile menu.  I need to talk to Seth about how to resolve that.  But take a look on staging please.

**Comment by MelissaBraxton** [comment]
LGTM!

**Comment by sadlerw** [comment]
@ASprinkle so with this new menu the fs admin users just need to navigate to any one of the admin urls (admin/christmas-trees/reports or season-dates or district-dates) and they get the menu which they can use for navigation to the others.  Based on that can you determine the answer to the task above: determine whether or not knowing the entry url is sufficient for fs admin users?  Is that enough? I'd be happy to discuss with you or others if you like.

**Comment by bryanmcfadden** [comment]
Updated task flow:

[admin_task_flow.pdf](https://github.com/nciinc/fs-permit-platform/files/1817175/admin_task_flow.pdf)

![image](https://user-images.githubusercontent.com/16689692/37492533-1c3bfe56-2878-11e8-8de8-627f3692c568.png)

**Comment by ASprinkle** [comment]
@sadlerw knowing the entry url is sufficient. We would likley want to share the one url out and let folks nav from there vs. sending out different or multiple urls.  Thanks, looks great!
